### PR TITLE
Logstash.install

### PR DESCRIPTION
**Pull request by JasonIamAUnixAdmin**
For BLUE-90

### CODE REVIEW

File: monitor/logstash/init.sls [hunk 1]
```diff
@@ -0,0 +1,26 @@
+# repo logstash
+logstash-repo:
+  pkgrepo:
+    - managed
+{% if grains["os_family"] == "Debian" %}
+    - key_url: https://packages.elasticsearch.org/GPG-KEY-elasticsearch
+    - name: deb https://packages.elastic.co/logstash/2.3/debian stable main
```
Comment (clarkperkins) [inline monitor/logstash/init.sls]: Do we want to add pillar data for choosing a version?

Comment (JasonIamAUnixAdmin) [inline monitor/logstash/init.sls]: Change made

File: monitor/logstash/init.sls [hunk 2]
```diff
@@ -0,0 +1,27 @@
+{%- set version = pillar.logstash.version -%}
```
Comment (clarkperkins) [inline monitor/logstash/init.sls]: `pillar.monitor.logstash.version`?

**Comment by clarkperkins** [comment]
👍

**Pull request merged**